src/sentry/organizations/services/organization/methods/list_organizations.py

```diff
@@ -0,0 +1,136 @@
+import logging
+from enum import Enum
+
+from django.db.models import Count, Q
+from rest_framework.response import Response
+
+from sentry.api.paginator import DateTimePaginator, OffsetPaginator
+from sentry.api.serializers import serialize
+from sentry.db.models.query import in_iexact
+from sentry.models.organization import Organization, OrganizationStatus
+from sentry.models.organizationmember import OrganizationMember
+from sentry.models.projectplatform import ProjectPlatform
+from sentry.search.utils import tokenize_query
+from sentry.users.services.user.service import user_service
+from sentry.utils.pagination_factory import PaginatorLike
+
+logger = logging.getLogger(__name__)
+
+
+class SortBy(Enum):
+    MEMBERS = "members"
+    PROJECTS = "projects"
+    DATE_ADDED = "date"
+
+
+class Show(Enum):
+    ALL = "all"
+
+
+def list_organizations(
```

Comment:
the API would call this RPC method, and if its fails remotely (e.g. on control), could try/catch onto calling local?